libs/coin-near/src/api/archive-node-sdk.ts

```diff
@@ -177,16 +174,18 @@ export const getStakingPositions = async (
   let totalAvailable = new BigNumber(0);
   let totalPending = new BigNumber(0);
 
+  const activeDelegatedStakeBalance = await account.getActiveDelegatedStakeBalance();
+
   const stakingPositions = await Promise.all(
-    stakingDeposits.map(async ({ validator_id: validatorId, deposit }) => {
+    activeDelegatedStakeBalance.stakedValidators.map(async ({ validatorId, amount: rawTotal }) => {
```

Comment:
No need to sanitize either `activeDelegatedStakeBalance` or `stakedValidators`?
(probably not but better safe than sorry)

```suggestion
    activeDelegatedStakeBalance?.stakedValidators?.map(async ({ validatorId, amount: rawTotal }) => {
```

Reply:
Based on the response even if there aren't any `stakedValidators` there will still be an empty array, but you're right - it wouldn't hurt adding optional chaining on that first prop `activeDelegatedStakeBalance?.stakedValidators` just in case :)